mock/py/mockbuild/scrub_all.py

```diff
@@ -0,0 +1,110 @@
+"""
+Logic for observing /var/lib/mock and /var/cache/mock, and try to cleanup as
+much as possible sub-directories there.
+"""
+
+
+import os
+from glob import glob
+import subprocess
+from mockbuild.constants import MOCKCONFDIR
+from mockbuild.config import traverse_chroot_configs
+
+
+def _do_scrub(configs, weird, chroot, suffix=None):
+    if suffix and chroot + "-" + suffix in weird:
+        print(f"skipping weird scrub: {chroot} {suffix}")
+        return
+
+    base_cmd = ["mock", "--scrub=all", "-r"]
```

Comment:
??? Now you can call the function directly. Or is there an obstacle?

Reply:
That would mean calling the following multiple times:

```python
commands = Commands(config_opts, uidManager, plugins, state, buildroot, bootstrap_buildroot)
commands.scrub(all) 
```
Which is _a lot_ of work (manpower, to code properly all the arguments)  :-/ my opinion is :shrug: keep it as is this is mostly cosmetics.  Admitedly that's not nice, but still better than nothing.

Reply:
Added the FIXME note at least.